.github/workflows/add-to-board.yml

```diff
@@ -21,12 +25,16 @@ jobs:
     env:
       PROJECT_ID_MAP: |
         {
-          "team:DocsEng":   "1625",
-          "team:Obs":       "649",
-          "team:Platform":  "1232",
-          "team:Projects":  "1415",
-          "team:Search":    "726",
-          "team:Security":  "1034"
+          "team:Ingest":      "649",
+          "team:Experience":  "1034",
+          "team:Admin":       "1232",
+          "team:Developer":   "1232",
```

Comment:
@georgewallace I set these to the Platform board for now. Feel free to update

Reply:
search board has been renamed to Developer. so just corrected it.